### ISSUE TITLE
note 

### ISSUE THREAD

**Issue by Hiradpi** [Gozargah/Marzban#706]
notes are not working for me is it normal that even after i define a note for a user it does not save it ?

```
{\"username\":\"manmozam\",\"note\":\"inKarNemikoneMashala\",\"proxies\":{\"vless\":{\"id\":\"1db15270-d036-4865-8474-e0e4e7c45864\",\"flow\":\"\"},\"vmess\":{\"id\":\"a2d79fd2-f883-434e-9633-e94ccde1e7ec\"}},\"data_limit\":0,\"expire\":null,\"data_limit_reset_strategy\":\"no_reset\",\"status\":\"active\",\"inbounds\":{\"vless\":[\"Vless` ws DE\",\"Vless ws2 IR\",\"VLESS TCP REALITY\",\"VLESS GRPC REALITY\"],\"vmess\":[\"VMess WS TR\",\"VMess WS LU\"]}}

```

**Comment by SaintShit** [comment]
if you mean inability of editing users note by `not working`, it's already fixed by 2f3bfde407af2aaad67b1ffbcb73f1f2aa99b03a on `dev` branch and gonna merged to latest version soon. otherwise, please provide more details.

**Comment by Hiradpi** [comment]
it is ? oh sorry 
but since you are here can you please consider adding the custom option for data_limit_reset_strategy
for example insted of year we give it a day int value like after 5 days or just client sends epoch value to the server insted of day , month etc ...
thank you
